site/ja/tutorials/images/hub_with_keras.ipynb

```diff
@@ -0,0 +1,1042 @@
+        "colab_type": "code",
+        "id": "rgXb44vt6goJ",
+        "colab": {}
+      },
+      "source": [
+        "predicted_class = np.argmax(result[0], axis=-1)\n",
+        "predicted_class"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "YrxLMajMoxkf"
+      },
+      "source": [
+        "### 予測結果をデコードする\n",
+        "\n",
+        "予測されたクラスの ID を「 ImageNet 」のラベルと突き合わせて、予測結果をデコードします。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "ij6SrDxcxzry",
+        "colab": {}
+      },
+      "source": [
+        "labels_path = tf.keras.utils.get_file('ImageNetLabels.txt','https://storage.googleapis.com/download.tensorflow.org/data/ImageNetLabels.txt')\n",
+        "imagenet_labels = np.array(open(labels_path).read().splitlines())"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "uzziRK3Z2VQo",
+        "colab": {}
+      },
+      "source": [
+        "plt.imshow(grace_hopper)\n",
+        "plt.axis('off')\n",
+        "predicted_class_name = imagenet_labels[predicted_class]\n",
+        "_ = plt.title(\"Prediction: \" + predicted_class_name.title())"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "amfzqn1Oo7Om"
+      },
+      "source": [
+        "## シンプルな転移学習"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "K-nIpVJ94xrw"
+      },
+      "source": [
+        "TF Hub を利用すると、私たちが用意したデータセット内のクラスを認識するために、モデルの最上位層を再トレーニングする事が容易になります。"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "Z93vvAdGxDMD"
+      },
+      "source": [
+        "### データセット\n",
+        "\n",
+        "この例では、TensorFlow の花データセットを使用します。"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "metadata": {
+        "colab_type": "code",
+        "id": "DrIUV3V0xDL_",
+        "colab": {}
+      },
+      "source": [
+        "data_root = tf.keras.utils.get_file(\n",
+        "  'flower_photos','https://storage.googleapis.com/download.tensorflow.org/example_images/flower_photos.tgz',\n",
+        "   untar=True)"
+      ],
+      "execution_count": 0,
+      "outputs": []
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "colab_type": "text",
+        "id": "jFHdp18ccah7"
+      },
+      "source": [
+        "このデータをモデルに読み込む最も簡単な方法は、「 tf.keras.preprocessing.image.ImageDataGenerator 」を使用することです。\n",
+        "\n",
+        "TensorFlow Hub のすべての画像モジュールは、「 [0、1] 」の範囲の float 入力を想定しています。これを実現するには「 ImageDataGenerator 」の「 rescale 」パラメータを使用してください。\n",
```

Comment:
ditto about `「」`.

[Strong Suggestion]
I think `[0、1]` should modified to `[0, 1]`.
`[x, y]` here is a mathematical notation which stand for [open interval](https://ja.wikipedia.org/wiki/%E5%8C%BA%E9%96%93_(%E6%95%B0%E5%AD%A6)) between 0 and 1.
In such case, mathematician use `,` though writing in Japanese.